<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLiveryFilterBar</class>
 <widget class="QFrame" name="CLiveryFilterBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>49</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Livery filter</string>
  </property>
  <layout class="QVBoxLayout" name="vl_LiveryFilterBar">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wi_Top" native="true">
     <layout class="QHBoxLayout" name="hl_Top">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="swift::gui::filters::CFilterBarButtons" name="filter_Buttons">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_LiveryCode">
        <property name="text">
         <string>Code:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_LiveryCode">
        <property name="placeholderText">
         <string>Livery code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_AirlineIcaoCode">
        <property name="text">
         <string>Airline:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_AirlineIcaoCode">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>ICAO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Description">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_Description"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_Bottom" native="true">
     <layout class="QHBoxLayout" name="hl_Bottom">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_Id">
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_Id">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Colors">
        <property name="text">
         <string>Color liveries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Airlines">
        <property name="text">
         <string>Airline liveries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Color">
        <property name="text">
         <string>Color (F/Tl):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::components::CColorSelector" name="color_Fuselage">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::components::CColorSelector" name="color_Tail">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hs_ColorDistance">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Color distance</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::filters::CFilterBarButtons</class>
   <extends>QFrame</extends>
   <header>gui/filters/filterbarbuttons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CColorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/colorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_LiveryCode</tabstop>
  <tabstop>le_AirlineIcaoCode</tabstop>
  <tabstop>le_Description</tabstop>
  <tabstop>le_Id</tabstop>
  <tabstop>cb_Colors</tabstop>
  <tabstop>cb_Airlines</tabstop>
  <tabstop>hs_ColorDistance</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
